src/website/app/demos/Menu/examples/menu/data.js

```diff
@@ -26,9 +26,11 @@ export default {
   title: 'Data-Driven',
   // $FlowFixMe
   backgroundColor: mineralTheme.color_gray_10,
-  description: `Menu content can also be defined by an array of data, with the
-    structure below. Object properties in the \`items\` array(s) will be passed
-    on to the [MenuItem](../menu-item).`,
+  description: `Menu content can also be defined by an array of data, with the structure shown in the code example below.
+Object properties in the \`items\` array(s) will be passed on to the [MenuItem](../menu-item).
+
+A new [MenuGroup](../menu-group) will be created for each array element that has an \`items\` property defined.
```

Comment:
Same comment applies here that I left on Dropdown's 'data' example.

Reply:
updated